index.html

```diff
@@ -0,0 +1,157 @@
+              <path d="M32.2666 21.2667L35.7333 6.86688" stroke="#EC5242" stroke-width="1.5" stroke-miterlimit="10" stroke-linecap="round" stroke-linejoin="round"/>
+              <path d="M54.6 21.3L57.7334 6.8335" stroke="#EC5242" stroke-width="1.5" stroke-miterlimit="10" stroke-linecap="round" stroke-linejoin="round"/>
+              <path d="M25.6667 40H52.3334" stroke="#EC5242" stroke-width="1.5" stroke-miterlimit="10" stroke-linecap="round" stroke-linejoin="round"/>
+              <path d="M22.3334 53.3333H49" stroke="#EC5242" stroke-width="1.5" stroke-miterlimit="10" stroke-linecap="round" stroke-linejoin="round"/>
+              </svg>              
+            <h4>Workshop</h4>
+            <p>Listen to speakers from various sectors about their professional fields.</p>
+          </li>
+          <li>
+            <svg width="80" height="80" viewBox="0 0 80 80" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <path d="M30.5333 36.2334C30.2 36.2 29.8 36.2 29.4333 36.2334C21.5 35.9667 15.2 29.4667 15.2 21.4667C15.2 13.3 21.8 6.66669 30 6.66669C38.1667 6.66669 44.8 13.3 44.8 21.4667C44.7667 29.4667 38.4667 35.9667 30.5333 36.2334Z" fill="#EC5242" stroke="white" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
+              <path d="M48.7 13.3333C55.1667 13.3333 60.3667 18.5666 60.3667 25C60.3667 31.3 55.3667 36.4333 49.1333 36.6666C48.8667 36.6333 48.5667 36.6333 48.2667 36.6666" fill="#EC5242"/>
+              <path d="M48.7 13.3333C55.1667 13.3333 60.3667 18.5666 60.3667 25C60.3667 31.3 55.3667 36.4333 49.1333 36.6666C48.8667 36.6333 48.5667 36.6333 48.2667 36.6666" stroke="white" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
+              <path d="M13.8667 48.5333C5.80001 53.9333 5.80001 62.7333 13.8667 68.1C23.0333 74.2333 38.0667 74.2333 47.2333 68.1C55.3 62.7 55.3 53.9 47.2333 48.5333C38.1 42.4333 23.0667 42.4333 13.8667 48.5333Z" fill="#EC5242" stroke="white" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
+              <path d="M56.1333 66.6667C58.5333 66.1667 60.8 65.2 62.6667 63.7667C67.8667 59.8667 67.8667 53.4334 62.6667 49.5334C60.8333 48.1334 58.6 47.2 56.2333 46.6667" fill="#EC5242"/>
+              <path d="M56.1333 66.6667C58.5333 66.1667 60.8 65.2 62.6667 63.7667C67.8667 59.8667 67.8667 53.4334 62.6667 49.5334C60.8333 48.1334 58.6 47.2 56.2333 46.6667" stroke="white" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
+            </svg>
+              
+            <h4>TLS Ignite</h4>
+            <p>Listen to speakers from various sectors about their professional fields.</p>
+          </li>
+        </ul>
+        <a href="#" class="see-main-program">SEE THE WHOLE PROGRAM</a>
+        <a href="#"><button>Join TL Summit 2023</button></a>
+      </section>
+
+      <section class="speakers"></section>
```

Comment:
According to the requirement, this section should be an article tag, kindly refactor it 😊 🟢